cpp/include/cudf/unary.hpp

```diff
@@ -31,6 +33,74 @@ namespace cudf {
  * @brief Column APIs for unary ops
  */
 
+/** @brief Convert a floating-point value to fixed-point
```

Comment:
Suggestion: please correct hyphenation

```suggestion
/** @brief Convert a floating-point value to fixed point
```

Alternatively: "Convert a floating-point value to a fixed-point value"